packages/ext-taquito/src/tzktReadProvider.ts

```diff
@@ -0,0 +1,169 @@
+import 'cross-fetch/polyfill';
+import {
+  BlockResponse,
+  EntrypointsResponse,
+  MichelsonV1Expression,
+  SaplingDiffResponse, ScriptedContracts,
+} from '@taquito/rpc';
+import {
+  BigMapQuery,
+  BlockIdentifier,
+  SaplingStateQuery,
+  TzReadProvider
+} from '@taquito/taquito';
+import {
+  accountsGetBalance,
+  accountsGetByAddress,
+  accountsGetCounter, bigMapsGetKey, Block, blocksGet,
+  blocksGetByHash,
+  blocksGetByLevel,
+  contractsGetEntrypoints,
+  contractsGetStorage,
+  contractsGetStorageSchema, headGet,
+  protocolsGetCurrent
+} from "@tzkt/sdk-api";
+import {BigNumber} from 'bignumber.js';
+
+
+export class TzktReadProvider implements TzReadProvider {
```

Comment:
Not implemented correctly, see See https://github.com/ecadlabs/taquito/blob/bbd425f610b7478f501ce34f771caa604f338593/packages/taquito/src/read-provider/interface.ts#L49

(missing block identifier params)